packages/hooks/src/useEventListener.ts

```diff
@@ -40,14 +40,34 @@ export default function useEventListener<Type extends keyof DocumentEventMap>(
       return;
     }
 
+    // @ts-expect-error Handle this in case non-TypeScript users pass in the wrong value
+    if (enabled === 'false') {
```

Comment:
Just for this case, not for `'true'`? `typeof enabled === 'string' && enabled !== 'once'` then we show a warning?

This will also throw some React warnings cuz it makes the `useEffect` conditional. I think that's fine, cuz it's better we do this than silently coerce it to a boolean.